module/applications/item/sheet-v2-mixin.mjs

```diff
@@ -171,7 +171,7 @@ export default function ItemSheetV2Mixin(Base) {
       // Properties
       context.properties = {
         active: [],
-        object: Object.fromEntries((context.source.properties ?? []).map(p => [p, true])),
+        object: Object.fromEntries((context.system.properties ?? []).map(p => [p, true])),
```

Comment:
Have to use the derived data here otherwise the "Magical Bonus" field won't appear on the sheet if the "Magical" property was added by an enchantment.